encoding/encoding.go

```diff
@@ -0,0 +1,78 @@
+package encoding
+
+import (
+	"bytes"
+	"encoding/binary"
+)
+
+const (
+	MaxObject         = 0x3FF
+	InstanceBits      = 22
+	MaxInstance       = 0x3FFFFF
+	MaxBitstringBytes = 15
+	ArrayAll          = 0xFFFFFFFF
+	NoPriority        = 0
+	MinPriority       = 1
+	MaxPriority       = 16
+)
+
+func EncodeUnsigned(value uint32) []byte {
+	switch {
+	case value < 0x100:
```

Comment:
Magic numbers. define them as consts if needed, leave comment where does they come from. If in standrad reference the standard and/or copy-pase necessary sections from the standrad to give the context to the reader.